2019-11-17 22:33:37.695 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:33:37.702 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:33:37.704 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:33:37.711 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:33:37.713 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:33:37.713 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:33:37.722 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:33:37.723 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:33:37.730 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:33:40.620 [error] <0.1638.0>@hackney_metrics:get_engine:26 gen_server tzdata_release_updater terminated with reason: bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26
2019-11-17 22:33:40.633 [error] <0.1638.0>@hackney_metrics:get_engine:26 CRASH REPORT Process tzdata_release_updater with 0 neighbours crashed with reason: bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26
2019-11-17 22:33:40.633 [error] <0.1632.0>@hackney_metrics:get_engine:26 Supervisor {<0.1632.0>,'Elixir.Supervisor.Default'} had child 'Elixir.Tzdata.ReleaseUpdater' started with 'Elixir.Tzdata.ReleaseUpdater':start_link() at <0.1638.0> exit with reason bad argument in call to ets:lookup_element(hackney_config, mod_metrics, 2) in hackney_metrics:get_engine/0 line 26 in context child_terminated
2019-11-17 22:36:23.208 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:36:23.214 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:36:23.216 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:36:23.221 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:36:23.222 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:36:23.223 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:36:23.228 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:36:23.230 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:36:23.269 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:37:00.002 [error] <0.361.0> ** Task <0.361.0> terminating
** Started from <0.360.0>
** When function  == #Fun<Elixir.Quantum.Executor.7.12665260>
**      arguments == []
** Reason for termination == 
** {'function not exported',[{'Elixir.Pubbot',test1,[[{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,5672}],<<"amqp.notificacao">>,<<>>,<<"templates/some_event.eex">>],[]},{'Elixir.Quantum.Executor','-run/4-fun-3-',3,[{file,"lib/quantum/executor.ex"},{line,141}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,"lib/task/supervised.ex"},{line,90}]},{'Elixir.Task.Supervised',reply,5,[{file,"lib/task/supervised.ex"},{line,35}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
2019-11-17 22:37:00.012 [error] <0.361.0>@Elixir.Pubbot:test1 CRASH REPORT Process <0.361.0> with 0 neighbours crashed with reason: call to undefined function 'Elixir.Pubbot':test1([{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,...}], <<"amqp.notificacao">>, <<>>, <<"templates/some_event.eex">>)
2019-11-17 22:37:00.012 [error] <0.350.0>@Elixir.Pubbot:test1 Supervisor 'Elixir.Pubbot.Scheduler.Task.Supervisor' had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.361.0> exit with reason call to undefined function 'Elixir.Pubbot':test1([{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,...}], <<"amqp.notificacao">>, <<>>, <<"templates/some_event.eex">>) in context child_terminated
2019-11-17 22:38:00.246 [error] <0.363.0> ** Task <0.363.0> terminating
** Started from <0.362.0>
** When function  == #Fun<Elixir.Quantum.Executor.7.12665260>
**      arguments == []
** Reason for termination == 
** {'function not exported',[{'Elixir.Pubbot',test1,[[{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,5672}],<<"amqp.notificacao">>,<<>>,<<"templates/some_event.eex">>],[]},{'Elixir.Quantum.Executor','-run/4-fun-3-',3,[{file,"lib/quantum/executor.ex"},{line,141}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,"lib/task/supervised.ex"},{line,90}]},{'Elixir.Task.Supervised',reply,5,[{file,"lib/task/supervised.ex"},{line,35}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
2019-11-17 22:38:00.246 [error] <0.363.0>@Elixir.Pubbot:test1 CRASH REPORT Process <0.363.0> with 0 neighbours crashed with reason: call to undefined function 'Elixir.Pubbot':test1([{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,...}], <<"amqp.notificacao">>, <<>>, <<"templates/some_event.eex">>)
2019-11-17 22:38:00.246 [error] <0.350.0>@Elixir.Pubbot:test1 Supervisor 'Elixir.Pubbot.Scheduler.Task.Supervisor' had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.363.0> exit with reason call to undefined function 'Elixir.Pubbot':test1([{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,...}], <<"amqp.notificacao">>, <<>>, <<"templates/some_event.eex">>) in context child_terminated
2019-11-17 22:38:28.018 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:38:28.031 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:38:28.035 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:38:28.042 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:38:28.044 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:38:28.044 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:38:28.050 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:38:28.052 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:38:28.102 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:41:01.000 [warning] <0.318.0> lager_error_logger_h dropped 7 messages in the last second that exceeded the limit of 50 messages/sec
2019-11-17 22:45:01.003 [warning] <0.318.0> lager_error_logger_h dropped 9 messages in the last second that exceeded the limit of 50 messages/sec
2019-11-17 22:46:30.193 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:46:30.213 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:46:30.224 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:46:30.236 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:46:30.239 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:46:30.240 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:46:30.252 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:46:30.256 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:46:30.327 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:46:52.584 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:46:52.590 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:46:52.592 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:46:52.598 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:46:52.599 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:46:52.599 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:46:52.605 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:46:52.607 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:46:52.653 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:47:00.003 [error] <0.359.0> ** Task <0.359.0> terminating
** Started from <0.358.0>
** When function  == #Fun<Elixir.Quantum.Executor.7.12665260>
**      arguments == []
** Reason for termination == 
** {'function not exported',[{'Elixir.Pubbot',publish,[[{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,5672}],<<"amqp.notificacao">>,<<>>,<<"templates/some_event.eex">>],[]},{'Elixir.Quantum.Executor','-run/4-fun-3-',3,[{file,"lib/quantum/executor.ex"},{line,141}]},{'Elixir.Task.Supervised',invoke_mfa,2,[{file,"lib/task/supervised.ex"},{line,90}]},{'Elixir.Task.Supervised',reply,5,[{file,"lib/task/supervised.ex"},{line,35}]},{proc_lib,init_p_do_apply,3,[{file,"proc_lib.erl"},{line,249}]}]}
2019-11-17 22:47:00.017 [error] <0.359.0>@Elixir.Pubbot:publish CRASH REPORT Process <0.359.0> with 0 neighbours crashed with reason: call to undefined function 'Elixir.Pubbot':publish([{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,...}], <<"amqp.notificacao">>, <<>>, <<"templates/some_event.eex">>)
2019-11-17 22:47:00.017 [error] <0.350.0>@Elixir.Pubbot:publish Supervisor 'Elixir.Pubbot.Scheduler.Task.Supervisor' had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.359.0> exit with reason call to undefined function 'Elixir.Pubbot':publish([{username,<<"guest">>},{password,<<"guest">>},{virtual_host,<<"/">>},{host,<<"localhost">>},{port,...}], <<"amqp.notificacao">>, <<>>, <<"templates/some_event.eex">>) in context child_terminated
2019-11-17 22:48:43.652 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:48:43.661 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:48:43.664 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:48:43.672 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:48:43.674 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:48:43.674 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:48:43.681 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:48:43.683 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:48:43.734 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:50:01.000 [warning] <0.318.0> lager_error_logger_h dropped 10 messages in the last second that exceeded the limit of 50 messages/sec
2019-11-17 22:54:23.412 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:54:23.419 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:54:23.422 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:54:23.426 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:54:23.428 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:54:23.428 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:54:23.433 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:54:23.435 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:54:23.479 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 22:55:00.202 [error] <0.324.0> ** Task <0.324.0> terminating
** Started from <0.321.0>
** When function  == #Fun<Elixir.Quantum.Executor.7.12665260>
**      arguments == []
** Reason for termination == 
** {#{'__exception__' => true,'__struct__' => 'Elixir.CompileError',description => <<"expected \"assigns\" to expand to an existing variable or be part of a match">>,file => <<"templates/hello.eex">>,line => 2},[{'Elixir.Kernel','var!',1,[{file,"expanding macro"}]},{elixir_compiler_0,'__FILE__',1,[{file,"templates/hello.eex"},{line,2}]}]}
2019-11-17 22:55:00.220 [error] <0.324.0> CRASH REPORT Process <0.324.0> with 0 neighbours crashed with reason: #{'__exception__' => true,'__struct__' => 'Elixir.CompileError',description => <<"expected \"assigns\" to expand to an existing variable or be part of a match">>,file => <<"templates/hello.eex">>,line => 2} in 'Elixir.Kernel':'var!'/1
2019-11-17 22:55:00.220 [error] <0.311.0> Supervisor 'Elixir.Pubbot.Scheduler.Task.Supervisor' had child undefined started with {'Elixir.Task.Supervised',start_link,undefined} at <0.324.0> exit with reason #{'__exception__' => true,'__struct__' => 'Elixir.CompileError',description => <<"expected \"assigns\" to expand to an existing variable or be part of a match">>,file => <<"templates/hello.eex">>,line => 2} in 'Elixir.Kernel':'var!'/1 in context child_terminated
2019-11-17 22:56:01.000 [warning] <0.279.0> lager_error_logger_h dropped 10 messages in the last second that exceeded the limit of 50 messages/sec
2019-11-17 22:57:40.542 [info] <0.43.0> Application lager started on node nonode@nohost
2019-11-17 22:57:40.550 [info] <0.43.0> Application xmerl started on node nonode@nohost
2019-11-17 22:57:40.553 [info] <0.43.0> Application jsx started on node nonode@nohost
2019-11-17 22:57:40.558 [info] <0.43.0> Application ranch started on node nonode@nohost
2019-11-17 22:57:40.561 [info] <0.43.0> Application recon started on node nonode@nohost
2019-11-17 22:57:40.561 [info] <0.43.0> Application rabbit_common started on node nonode@nohost
2019-11-17 22:57:40.570 [info] <0.43.0> Application amqp_client started on node nonode@nohost
2019-11-17 22:57:40.573 [info] <0.43.0> Application amqp started on node nonode@nohost
2019-11-17 22:57:40.632 [info] <0.43.0> Application pubbot started on node nonode@nohost
2019-11-17 23:00:01.003 [warning] <0.288.0> lager_error_logger_h dropped 18 messages in the last second that exceeded the limit of 50 messages/sec
